Rakefile

```diff
@@ -23,7 +23,9 @@ RSpec::Core::RakeTask.new(:spec) do |spec|
 end
 
 desc 'Run RuboCop over this gem'
-RuboCop::RakeTask.new(:internal_investigation)
+RuboCop::RakeTask.new(:internal_investigation) do
+  RuboCop::ConfigLoader.inject_defaults!("#{__dir__}/config/default.yml")
+end
```

Comment:
I don’t understand why this is necessary, and hope to remove it before merging the PR.